lsp/nls/src/background.rs

```diff
@@ -147,7 +179,11 @@ impl SupervisorState {
         let mut tx = tx.ok_or_else(|| anyhow!("failed to get worker stdin"))?;
         let mut rx = rx.ok_or_else(|| anyhow!("failed to get worker stdout"))?;
 
-        bincode::serde::encode_into_std_write(eval, &mut tx, bincode::config::standard())?;
+        bincode::serde::encode_into_std_write(
+            eval.eval_ref(),
```

Comment:
Is `eval_ref` used to serialize exactly as `EvalCommand` but without requiring ownership?